lib/config/config-schema.json

```diff
@@ -45,6 +45,19 @@
             "required": [
                 "host"
             ]
+        },
+        "whitelist": {
```

Comment:
I need to describe this in the README

Reply:
In general, why do you use a config file for this? This should work like this:
1. We executed the tcm-export: case command and got the result in TCM for the report with the key in "case";
2. We executed the tcm-export: describe command - and got the result in TCM for the report with the key in the "describe"
3. We ran the tcm-export: describeversio command (this is just an example), it is "node ./bin --target describe --cycleName testdecribe --checkVersion" (and this is also an example) and already the option --checkVersion responsible for checking the version in the test